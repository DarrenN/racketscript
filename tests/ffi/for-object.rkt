#lang racketscript/base

(require racketscript/interop)

(#js*.console.log (for/js-object ([i (in-range 10)]
                                  [j (in-range 10)])
                    (values (+ i j) (* i j))))
