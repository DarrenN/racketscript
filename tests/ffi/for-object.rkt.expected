{
  '0': 0,
  '2': 1,
  '4': 4,
  '6': 9,
  '8': 16,
  '10': 25,
  '12': 36,
  '14': 49,
  '16': 64,
  '18': 81
}
